src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala

```diff
@@ -84,19 +76,18 @@ final class MobilitySimulator(
       s"Simulation triggered with tick: $tick -- Current time: $currentTime"
     )
 
-    /* Receive available charging points of evcs from SIMONA */
-    val availableChargingPoints: Map[UUID, Int] = {
-      evData.requestAvailablePublicEvCs().asScala.toMap.map { case (k, v) =>
-        (k, v.intValue())
+    /* Receive available charging points of evcs from SIMONA and converting them to scala values */
+    val availableChargingPoints =
+      evData.requestAvailablePublicEvCs().asScala.toMap.map {
+        case (cs, freeLots) =>
+          (cs, freeLots.intValue())
       }
-    }
 
-    /* Receive current prices for public evcs situation */
-    val currentPricesAtChargingStations: Map[UUID, Double] = {
-      evData.requestCurrentPrices().asScala.toMap.map { case (k, v) =>
-        (k, v.doubleValue())
+    /* Receive current prices for public evcs situation and converting them to scala values */
+    val currentPricesAtChargingStations =
+      evData.requestCurrentPrices().asScala.toMap.map { case (cs, prices) =>
```

Comment:
Not to be nitpicky here, but it's _one_ price, not multiple, right? :)

Reply:
You are right, I will change this name.